api/src/db/dynamoUserDataMigration.test.ts

```diff
@@ -48,10 +49,12 @@ describe("DynamoUserDataClient Migrations", () => {
 
   let client: DynamoDBDocumentClient;
   let dynamoUserDataClient: UserDataClient;
+  let logger: LogWriterType;
 
   beforeEach(async () => {
     client = DynamoDBDocumentClient.from(new DynamoDBClient(config), dynamoDbTranslateConfig);
-    dynamoUserDataClient = DynamoUserDataClient(client, dbConfig.tableName);
+    logger = LogWriter("NavigatorWebService", "DataMigrationLogs", "us-test-1");
```

Comment:
We should check that this isn't actually logging to AWS, or see if we can mock the logger.

Reply:
I used the DummyLogWriter Instead